extensions/mssql/src/mssql.d.ts

```diff
@@ -409,10 +323,6 @@ declare module 'mssql' {
 	export interface ValidateStreamingJobResult extends azdata.ResultStatus {
 	}
 
-	export interface ParseTSqlScriptResult {
```

Comment:
Make sure you're merged up to the latest from main and aren't removing any local changes you made for other stuff

Reply:
Looks like this was deleted in main and seems added here when merged on main! I frequently merge latest main before every iteration i add.